invenio_rdm_records/records/providers/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
just wondering about the module name, reading the path:
`/records/providers/...` should we have instead `/records/pidproviders/...` ?

Reply:
Do we forsee having minters, resolvers, etc.? If yes, what about `/records/pids/providers/` allowing for `..../minters`, `.../resolvers`, etc.

Reply:
at this time I don't think so, but you could anyway have `pids/providers`! :)